tests/nnfw_api/src/one_op_tests/While.test.cc

```diff
@@ -265,6 +265,9 @@ TEST_F(GenModelTest, neg_while_wrong_dtype)
   tc.expectFailRun();
```

Comment:
If test fail on compile phase, we can remove `tc` variable and `tc.expectFailRun()` call.

Reply:
OK. I will remove it.